src/Paramore.Brighter.MessagingGateway.AWSSQS/ValidateTopicByArn.cs

```diff
@@ -34,9 +34,19 @@ public class ValidateTopicByArn : IDisposable, IValidateTopic
     {
         private AmazonSimpleNotificationServiceClient _snsClient;
 
-        public ValidateTopicByArn(AWSCredentials credentials, RegionEndpoint region)
+        public ValidateTopicByArn(AWSCredentials credentials, RegionEndpoint region, Action<ClientConfig> clientConfigAction = null)
         {
-            _snsClient = new AmazonSimpleNotificationServiceClient(credentials, region);
+            var config = new AmazonSimpleNotificationServiceConfig
```

Comment:
I wonder if we should follow the pattern elsewhere of encapsulating this in a CreateSnsClient method. I like symmetry but its just an opinion